harvester_e2e_tests/integration/test_backup_restore.py

```diff
@@ -132,7 +132,19 @@ def config_backup_target(api_client, conflict_retries, backup_config):
     for backup in data['data']:
         endpoint = backup['status']['backupTarget'].get('endpoint')
         if endpoint != origin_spec.value.get('endpoint'):
-            api_client.backups.delete(backup['metadata']['name'])
+            target_backup = backup['metadata']['name']
+            api_client.backups.delete(target_backup)
+            endtime = datetime.now() + timedelta(seconds=wait_timeout)
+            while endtime > datetime.now():

```

Comment:
the check condition should move out of for-loop.
otherwise the maximum will multiply with wait_timeout as `backups * wait_timeout`

so the expected change would like:
- delete all backups
- check deleted backups is unavailable

Not:
- delete all backups
   - delete backup and check it is unavailable